include/exadg/operators/operator_base.cpp

```diff
@@ -744,7 +743,8 @@ OperatorBase<dim, Number, n_components>::init_system_matrix(
   dealii::TrilinosWrappers::SparseMatrix & system_matrix,
   MPI_Comm const &                         mpi_comm) const
 {
-  internal_init_system_matrix(system_matrix, mpi_comm);
+  dealii::DynamicSparsityPattern dsp;
```

Comment:
Why is the variable `dsp` needed outside the function call?

Reply:
We need it for `dealii::SparseMatrixTools::restrict_to_full_matrices`. I have marked the usage below.